front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/Permissions.groovy

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.front50.model
+
+import com.netflix.spinnaker.fiat.model.Authorization
+import groovy.transform.InheritConstructors
+
+@InheritConstructors
+class Permissions extends HashMap<Authorization, List<String>> {
```

Comment:
Does it make sense to move Permissions into fiat-api instead of duplicating it here and clouddriver?